test/unit/lib/choicesOverview.js

```diff
@@ -0,0 +1,14 @@
+const choicesOverview = require('../../../app/lib/choicesOverview');
+const chai = require('chai');
+
+const expect = chai.expect;
+
+describe('choicesOverview', () => {
+  it('should have valid ID appendend to the URL', () => {
```

Comment:
This test doesn't test each value in the input array is returned correctly. Please add another test for this scenario.

Reply:
Done

Reply:
There is still no check for multiple values in the input list returning different values for `choicesOverviewUrl`. This leaves the implementation possibly being changed and the test not picking up on it.

Reply:
Done